core/src/test/java/com/datastrato/gravitino/storage/kv/TestRocksDBKvBackend.java

```diff
@@ -161,6 +161,79 @@ void testDeleteRange() throws IOException {
     Assertions.assertNotNull(kvBackend.get("accb".getBytes(StandardCharsets.UTF_8)));
   }
 
+  @Test
+  void testDeleteRange_isStartExclusiveTrue() throws IOException {
```

Comment:
Can the name be shortened or made more concise? Character `_` seems to be not very elegant in Java.